sync/src/types/mod.rs

```diff
@@ -100,9 +100,22 @@ impl ChainSyncState {
         self.headers_sync_state = HeadersSyncState::Suspend(until)
     }
 
+    fn already_sync(&mut self) {
+        let now = unix_time_as_millis();
+        // 28 seconds
```

Comment:
suggest to add comment `use avg block interval: (MAX_BLOCK_INTERVAL + MIN_BLOCK_INTERVAL) / 2 = 28`